jukebox_radio/streams/models/queue_interval_model.py

```diff
@@ -18,27 +18,35 @@ def serialize(self, queue_interval):
             "lowerBound": Marker.objects.serialize(queue_interval.lower_bound),
             "upperBound": Marker.objects.serialize(queue_interval.upper_bound),
             "isMuted": queue_interval.is_muted,
-            "repeatCount": queue_interval.repeat_count,
         }
 
     def create_queue_interval(
```

Comment:
some small bugfixes to missing cases here. the diff here is ugly, but the code is clearer now (especially when you look at the logic and methods being called.)